client-core/src/service/wallet_state_service.rs

```diff
@@ -411,6 +411,12 @@ enum MementoOperation {
 }
 
 impl WalletStateMemento {
+    /// get length of memento
+    #[inline]
+    pub fn get_length(&self) -> usize {
```

Comment:
```suggestion
    pub fn len(&self) -> usize {
```

keep the naming convention with std lib?

Reply:
Clippy will also complain about `is_empty`.

Reply:
ok, i'll change to len
and fix clippy issue